src/Sulu/Bundle/MediaBundle/Tests/Functional/SearchIntegration/SearchIntegrationTest.php

```diff
@@ -104,8 +104,8 @@ public function testIndex($format, $expectedException)
         $this->documentManager->flush();
 
         $documents = $testAdapter->getDocuments();
-        $this->assertCount(1, $documents);
-        $document = current($documents);
+        $this->assertTrue(count($documents) >= 1);
+        $document = end($documents);
```

Comment:
i think this comes with the dump of the webspace nodes. but to be honest i have to investigate why i have done this